common/src/main/java/com/microsoft/identity/common/internal/fido/WebAuthnJsonUtil.kt

```diff
@@ -67,9 +68,13 @@ class WebAuthnJsonUtil {
          * @param fullResponseJson AuthenticationResponse Json string.
          */
         fun extractAuthenticatorAssertionResponseJson(fullResponseJson : String): String {
-            return JSONObject(fullResponseJson)
+            val fullResponseJsonObject = JSONObject(fullResponseJson);
+            val authResponseJsonObject = fullResponseJsonObject
                 .getJSONObject(FidoConstants.WEBAUTHN_AUTHENTICATION_ASSERTION_RESPONSE_JSON_KEY)
-                .toString()
+                // Making sure that Id is here because ESTS expects it.
+                // I've noticed that GPM will sometimes not include the id in the response object.
```

Comment:
google's bug? should we file it?

Reply:
Once folks get back from OOF, I'm going to ask internally if this is a bug or not. I'm looking through official Webauthn documentation, and I'm not finding concrete info on id as a field in authentication assertion responses (though it is clearly documented as a field in the outer JSON object).